### PR TITLE
Adds postalcode rules and translations for each different country

### PR DESCRIPTION
**Pull request by lucasayb**
Changes made:
Add the missing country rules for some countries

Goal of this change:
Without some translations, the checkout keeps returning error, which causes the Google Maps Form not to load. The goal of this PR is not to apply the translations, but to mitigate this error by now and then later do the correct translations

### CODE REVIEW

**Comment by lucasayb** [comment]
This PR is related with this one: https://github.com/vtex-apps/checkout-ui-custom/pull/101

**Comment by lucasayb** [comment]
Any news about this? It's urgent!

**Review by albertm805 — CHANGES_REQUESTED** [review]
This is a compiled file. We need to add the original file where the changes were applied.

File: checkout-ui-custom/src/_js/_countries.js [hunk 1]
```diff
@@ -192,6 +192,20 @@ module.exports._countriesrules = {
     number: false,
     postalCode: true,
   },
+  CHE: {
+    state: 'administrative_area_level_1',
+    city: 'locality',
+    number: !1,
+    postalCode: !0,
```
Comment (albertm805) [inline checkout-ui-custom/src/_js/_countries.js:199]: Why is this set with a `!`?

File: checkout-ui-custom/src/_js/_countries.js [hunk 2]
```diff
@@ -192,6 +192,20 @@ module.exports._countriesrules = {
...
+    postalCodeLength: 4,
+  },
+  ROU: {
+    state: 'administrative_area_level_1',
+    city: 'locality',
+    number: !1,
```
Comment (albertm805) [inline checkout-ui-custom/src/_js/_countries.js:205]: Here also

File: checkout-ui-custom/src/_js/_countries.js (hunk 1 above)
Comment (lucasayb) [reply checkout-ui-custom/src/_js/_countries.js:199]: I assumed it was to allow only numbers.

File: checkout-ui-custom/src/_js/_countries.js (hunk 2 above)
Comment (albertm805) [reply checkout-ui-custom/src/_js/_countries.js:205]: This has to be Boolean

**Comment by albertm805** [comment]
PR closed, as this change was merged in another PR

**Pull request closed without merging**